riff-raff/app/controllers/authentication.scala

```diff
@@ -104,22 +104,7 @@ class Login(
   }
 
   def oauth2Callback = Action.async { implicit request =>
-    import cats.instances.future._
-    (for {
-      identity <- checkIdentity()
-      _ <- EitherT.fromEither[Future] {
-        if (validator.isAuthorised(identity)) Right(())
```

Comment:
This [functionality](https://github.com/guardian/riff-raff/blob/23ec481b47d122f17da43ebe5b44be0903ce07af/riff-raff/app/controllers/authentication.scala#L49-L67) is checking the allowlist of email addresses _and_ the domain. We want to drop the former but not the latter, but Play Google Auth [does the domain checks now](https://github.com/guardian/play-googleauth/blob/c2fa7e703d3d122b84626cb7147ef7cd2283964a/play-v29/src/main/scala/com/gu/googleauth/auth.scala#L222-L225) thanks to [this config](https://github.com/guardian/riff-raff/blob/39c7e0dcaba44aad4588a11bfbdee8e0b0e3e8c4/riff-raff/app/AppComponents.scala#L227), so I don't think there is any need to repeat it here.